src/main/java/com/datadoghq/connect/logs/sink/DatadogLogsSinkConnectorConfig.java

```diff
@@ -33,6 +33,7 @@ public class DatadogLogsSinkConnectorConfig extends AbstractConfig {
     private static final String DD_URL_FORMAT_FROM_SITE = "http-intake.logs.%s:443";
     private static final String DEFAULT_DD_SITE = "datadoghq.com";
     public static final String DEFAULT_DD_URL = String.format(DD_URL_FORMAT_FROM_SITE, DEFAULT_DD_SITE);
+    public static final String USE_RECORD_TIMESTAMP = "datadog.use.record.timestamp";
```

Comment:
Could we call this `datadog.add_published_date`? I think that's a bit clearer. I'm not sure we'll end up with additional options in a `datadog.use.record` namespace where it'd be make sense to have an option namespaced like that.

Reply:
changed the parameter name to datadog.add_published_date